internal/component/loki/source/docker/docker.go

```diff
@@ -305,9 +305,10 @@ func (c *Component) getManagerOptions(args Arguments) (*options, error) {
 	}
 
 	return &options{
-		client:    client,
-		handler:   loki.NewEntryHandler(c.handler.Chan(), func() {}),
-		positions: c.posFile,
+		client:              client,
+		handler:             loki.NewEntryHandler(c.handler.Chan(), func() {}),
+		positions:           c.posFile,
+		taskRestartInterval: 5 * time.Second,
```

Comment:
hardcoded to 5s because I don't think that it's an interesting param to expose to the user